coderd/oauth2.go

```diff
@@ -234,7 +234,7 @@ func (api *API) deleteOAuth2ClientConfiguration() http.HandlerFunc {
 
 // @Summary OAuth2 device authorization request (RFC 8628).
 // @ID oauth2-device-authorization-request
-// @Accept json
+// @Accept application/x-www-form-urlencoded
 // @Produce json
```

Comment:
I found it hard to believe, but here it is https://datatracker.ietf.org/doc/html/rfc8628#section-3.2